examples/stats/multiple_wragged_outputs.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-output model with wragged time measurements\n",
+    "\n",
+    "This notebook describes how to fit a model in PINTS which has multiple outputs with different measurement times for each. Additionally, we illustrate how this same framework allows different measurement models to be used for different outputs within the same forward model when doing inference.\n",
+    "\n",
+    "For this, we consider the [Goodwin-Oscillator](../toy/model-goodwin-oscillator.ipynb) model, which has three outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pints\n",
+    "import pints.toy as toy\n",
+    "import pints.plot\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pints.noise"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We first create some simulated data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = pints.toy.GoodwinOscillatorModel()\n",
+    "real_parameters = model.suggested_parameters()\n",
+    "times = model.suggested_times()\n",
+    "values = model.simulate(real_parameters, times)\n",
+    "\n",
+    "# add noise\n",
+    "noise1 = 0.001\n",
+    "noise2 = 0.01\n",
+    "noise3 = 0.1\n",
+    "noisy_values = np.array(values, copy=True)\n",
+    "noisy_values[:, 0] += np.random.normal(0, noise1, len(times))\n",
+    "noisy_values[:, 1] += np.random.normal(0, noise2, len(times))\n",
+    "noisy_values[:, 2] += np.random.normal(0, noise3, len(times))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we assume that only the first half of time measurements are taken for the first two outputs; and only the second half for the third."
```

Comment:
Can we make this a bit less artificial? I.e. start with wordy version, saying e.g. we measure X every 6 hours, but Y only once a day. Then show how that leads to code like given below?